README.md

```diff
@@ -87,11 +87,11 @@ from for pull requests.
 ~~~ yaml
 approvers:
     - name1
-    - name2
+    - name2 email-address or any comment
```

Comment:
why not an optional field? 
``` yaml
approvers
    - name1
    - name2
      email: <mail>
```

At least it will remain valid yaml.
Could potentially add the `lead` field as well? (Or something better named than *lead*)

Added benefit this works throughout the OWNERS structure

Reply:
ok. I can do.

NOTE: we need also to fit the 'team lead" info. 
So I add a second field : 

```
- miekg
  email: xxxx
  team-lead-end-period: <date>
```